tests/11register.pl

```diff
@@ -140,3 +140,56 @@
          Future->done( 1 );
       });
    };
+
+test "registration remembers parameters",
+   requires => [ $main::API_CLIENTS[0] ],
+
+   do => sub {
+      my ( $http ) = @_;
+
+      my $desired_localpart = "11register_remember";
```

Comment:
Hmm. I was about to suggest using the fixture for providing this, except there isn't one yet. :/